classes/entity.py

```diff
@@ -52,6 +52,9 @@ def add_fields_to_name(self, fields):
     def _generated_name(self):
         return "%s-%s" % (self.type, self.properties['id'])
 
+    def __getitem__(self, key):
+        return self.properties.get(key, "BLANK")
```

Comment:
This feels like an odd default.  At minimum I would make it a constant and if it has to be not be None, then I would make it a little more distinct.  Maybe "<BLANK>".  "<NULL>" would be more natural to me than "<BLANK>", but that's a matter of personal preference.